csrankings-h.csv

```diff
@@ -941,6 +941,7 @@ Huy Phan,Queen Mary University of London,https://pquochuy.github.io,RegoACcAAAAJ
 Huzefa Rangwala,George Mason University,https://cs.gmu.edu/~hrangwal,yWJ9BqEAAAAJ
 Huzur Saran,IIT Delhi,http://www.cse.iitd.ernet.in/~saran,NOSCHOLARPAGE
 Hwa-Chun Lin,National Tsing Hua University,http://alpha5.cs.nthu.edu.tw/labweb/advisor.html,NOSCHOLARPAGE
+Hwajung Hong,KAIST,http://hwajunghong.com,W6dBWJkAAAAJ

```

Comment:
Please re-open with evidence that this faculty member (in Industrial Design) can solely advise PhD students for a degree in Computer Science.